### DISCUSSION TITLE
Global TTL

### DISCUSSION THREAD

**Discussion by ItsMrCube** [diced/zipline#237] [Ideas]
Storage can get expensive or run out over time when not managed correctly. I propose to have a global [TTL](https://en.wikipedia.org/wiki/Time_to_live) setting implemented.
The setting would allow admins to make all existing and future uploads ephemeral based on a user-defined duration.

**Comment by diced** [comment]
Added in https://github.com/diced/zipline/commit/78e884e97ef054b130482dd44334fe4e63519dff. Available via the `UPLOADER_DEFAULT_EXPIRATION` variable, and supports the same values that work in the `Expiry` header.
